website/static/css/pages/project-page.css

```diff
@@ -142,3 +142,7 @@
 span#nodeDescriptionEditable{
     white-space:pre-line;
 }
+
+#render-node  .ball-scale>div {
+    background-color: #BED1E2;
```

Comment:
Is there a way to DRY this up with https://github.com/CenterForOpenScience/osf.io/pull/5523/files/9b4687efef370dc11e0bbe678c0fb3fa0dab6231#r62506263 ?